app.rb

```diff
@@ -0,0 +1,112 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books(_books)
+    @books.each_with_index do |book, index|
+      puts "#{index + 1}) Title: \"#{book.title}\", Author: #{book.author}"
+    end
+    puts ''
+  end
+
+  def list_people(_people)
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+    puts ''
+  end
```

Comment:
While listing the people, it is displaying wrong info about the teacher. Instead of displaying the name, it is displaying the specialization instead of the name. ⬇️
![image](https://github.com/shihabrafio/OOP-School-Library/assets/33268927/c39b7048-4601-47e9-a912-53216d887ae2)